servers/rendering_server.h

```diff
@@ -1090,10 +1099,17 @@ class RenderingServer : public Object {
 		SCENARIO_DEBUG_SHADELESS,
 	};
 
+	enum ScenarioOcclusionCullingBuildQuality {
+		SCENARIO_OCCLUSION_BUILD_QUALITY_LOW = 0,
+		SCENARIO_OCCLUSION_BUILD_QUALITY_MEDIUM = 1,
+		SCENARIO_OCCLUSION_BUILD_QUALITY_HIGH = 2,
+	};
+
 	virtual void scenario_set_debug(RID p_scenario, ScenarioDebugMode p_debug_mode) = 0;
 	virtual void scenario_set_environment(RID p_scenario, RID p_environment) = 0;
 	virtual void scenario_set_fallback_environment(RID p_scenario, RID p_environment) = 0;
 	virtual void scenario_set_camera_effects(RID p_scenario, RID p_camera_effects) = 0;
+	virtual void scenario_set_occlusion_culling_build_quality(RID p_scenario, ScenarioOcclusionCullingBuildQuality p_quality) = 0;
```

Comment:
Again would make it single API for all viewports, no need to tweak per viewport and this value will most likely never change from high.

Reply:
This makes me think, should we have a setting to turn off occlusion culling for a viewport entirely? For instance, if you use a viewport to render an overview map.

Also, does occlusion culling work with an orthographic camera, or does it get automatically disabled in this case?